### ISSUE TITLE
No `key` prop warning

### ISSUE THREAD

**Issue by burtek** [parsable/react-truncate-markup#7]
I got the error `Warning: Each child in an array or iterator should have a unique "key" prop. See https://fb.me/react-warning-keys for more information.` while using `react-truncate-markup` with custom ellipsis being callback function. I attach the component causing the problem.

[issue.tsx.zip](https://github.com/parsable/react-truncate-markup/files/2145202/issue.tsx.zip)

**Comment by patrik-piskay** [comment]
Is it possible that your `props.text` is causing this warning?

**Comment by burtek** [comment]
they are plain strings, so I guess not, I might be wrong though.

What I found out, is that this error appears when the text is truncated. `renderTruncatedEllipsis` is the ellipsis callback which returns `React.Fragment`
![image](https://user-images.githubusercontent.com/2940958/42046584-79d490b8-7afe-11e8-83f3-f51d1e9eab10.png)

But this `React.Fragment` is rendered by `react-truncate-markup` as array:

![image](https://user-images.githubusercontent.com/2940958/42046848-0fd3db0a-7aff-11e8-9447-b4bb05bea375.png)

If I'm correct, the problem might be caused by putting everything in arrays [somewhere around here](https://github.com/parsable/react-truncate-markup/blob/master/src/index.js#L240)

**Comment by burtek** [comment]
The full error I get is:

```
Warning: Each child in an array or iterator should have a unique "key" prop. See https://fb.me/react-warning-keys for more information.
    in span
    in TruncateMarkup (created by ExpandableTruncate)
    in ExpandableTruncate (created by BigHeader)
    in (...)
```

**Comment by patrik-piskay** [comment]
I see, it looks like it has something to do with the fragment usage. I'll what we can do about it

**Comment by burtek** [comment]
Sure. Both `react` and `react-dom` exactly `16.4.1`, `react-truncate-markup: 1.1.1`. Checked our `yarn.lock` to be sure

**Comment by burtek** [comment]
Also, we're using typescript 2.9.1 currently in our project

**Comment by burtek** [comment]
Sorry, wrong butotn

**Comment by burtek** [comment]
Any plans for releasing this one? 🤔

**Comment by patrik-piskay** [comment]
Released in 1.1.2
